components/com_kunena/site/template/crypsis/layouts/message/item/default.php

```diff
@@ -32,7 +32,13 @@
 <div class="badger-left badger-info <?php if ($message->getAuthor()->isModerator()) : ?> badger-moderator <?php endif;?>"
 	 data-badger="<?php echo (!$isReply) ? $this->escape($avatarname) . ' created the topic: ' : $this->escape($avatarname) . ' replied the topic: '; ?><?php echo $message->displayField('subject'); ?>">
 	<div class="kmessage">
-		<div class="kmsg"> <?php echo $message->displayField('message'); ?> </div>
+		<p class="kmsg">
+			<?php  if (!$this->me->userid && !$isReply) :
```

Comment:
These if statements are more complicated than they need to be.

    if (this->config->teaser && $isReply && !$this->me->userid) { teaser } else { message }

Another question is if we should hide the whole message and not just contents...